libqtile/drawer.py

```diff
@@ -105,8 +105,9 @@ def font_family(self):
 
     @font_family.setter
     def font_family(self, font):
-        d = self.fontdescription()
-        d.set_family(font)
+        # d = self.fontdescription()
+        d = pangocffi.FontDescription.from_string(font)
+        # d.set_family(font)
```

Comment:
d.set_family(font) do not work then bold font like 'Times Bold' - groupbox widget stays with default font.